### PR TITLE
[WIP] combo-free Sleeping Carp

### PR DESCRIPTION
**Pull request by hornygranny**
this is just an example pending my other PR, that shows how a martial art could work without using clumsy combos. i wouldn't merge this as it's pretty savagely overpowered in this state, but it's fun

### CODE REVIEW

File: code/datums/martial.dm [hunk 1]
```diff
@@ -409,11 +381,11 @@
...
 	usr << "<b><i>You retreat inward and recall the teachings of the Sleeping Carp...</i></b>"
 
-	usr << "<span class='notice'>Wrist Wrench</span>: Disarm Disarm. Forces opponent to drop item in hand."
-	usr << "<span class='notice'>Back Kick</span>: Harm Grab. Opponent must be facing away. Knocks down."
-	usr << "<span class='notice'>Stomach Knee</span>: Grab Harm. Knocks the wind out of opponent and stuns."
-	usr << "<span class='notice'>Head Kick</span>: Disarm Harm Harm. Decent damage, forces opponent to drop item in hand."
-	usr << "<span class='notice'>Elbow Drop</span>: Harm Disarm Harm Disarm Harm. Opponent must be on the ground. Deals huge damage, instantly kills anyone in critical condition."
+	usr << "<span class='notice'>Wrist Wrench</span>: Disarm intent. Forces opponent to drop item in hand, stuns and hurts slightly."
+	usr << "<span class='notice'>Back Kick</span>: Harm intent while facing away. Knocks target down and does damage."
+	usr << "<span class='notice'>Groin Knee</span>: Harm intent on groin. Knocks the wind out of opponent, stuns, and damages."
```
Comment (Iamgoofball) [inline code/datums/martial.dm]: this is ten times funnier now

**Comment by PKPenguin321** [comment]
We could just go the Goon way and make it like the wrestling belt where it gives you a bunch of action buttons with cooldowns

**Comment by hornygranny** [comment]
I think for more abstract styles like plasma fist that will be needed.

**Comment by AlexanderUlanH** [comment]
Could be balanced by replacing combos with combo points.
Heck, the action buttons could be replaced with combo points.
Harm intent hits are unspectacular and build a point per hit, disarm executes the most expensive move possible with the points you've accumulated.
For sleeping carp: 0 points gets you a wrist wrench, 1 point gets you a back kick if possible or a stomach knee, two gets you an elbow drive if possible, or a head kick.

**Comment by KorPhaeron** [comment]
> i wouldn't merge this 

Okay

**Comment by KorPhaeron** [comment]
I'd rather just remove sleeping carp entirely than turn it into "you have 4 different infinite harm batons built into your hands." You seem to be doing a more fun/admin only version in another PR anyway, and think this stuff isn't merge worthy by your own admission

**Pull request closed without merging**